pkg/serializer/internal/metrics/iterable_series.go

```diff
@@ -63,7 +63,7 @@ func (series IterableSeries) MarshalJSON() ([]byte, error) {
 	// use an alias to avoid infinite recursion while serializing a Series
 	type SeriesAlias Series
 
-	var seriesAlias SeriesAlias
```

Comment:
I've made this kind of mistake a few times -- maybe this instance is worth a quick unit test so it doesn't regress on the next refactor?

Reply:
Unit test added: `TestIterableSeriesEmptyMarshalJSON`